modules/administration-guide/partials/proc_enabling-self-registration.adoc

```diff
@@ -0,0 +1,14 @@
+// configuring-authorization
+
+[id="enabling-self-registration_{context}"]
+= Enabling self-registration
```

Comment:
Is there a topic in the docs that would explain to me what `self-registration` is?
If that's something that a person, who is reading this doc, should already know - that's okay.
But as a reviewer and not being an expert in an administration part - I'm not sure I completely understand what is `self-registration`.

Reply:
@azatsarynnyy I've added a few sentences on the introduction that explains what self-registration is used for https://github.com/eclipse/che-docs/pull/1819/commits/305535f261b7ee9761ee1dd0c2886ad3e3d40973 PTAL